components/Card.js

```diff
@@ -11,8 +11,10 @@ export const CardElement = styled.article`
 
 export default function Card({ text }) {
   return (
-    <CardElement>
-      <p>{text}</p>
-    </CardElement>
+    <>
+      <CardElement>
+        <p>{text}</p>
+      </CardElement>
+    </>
```

Comment:
you don't need fragments here:

```suggestion
      <CardElement>
        <p>{text}</p>
      </CardElement>
```